### DISCUSSION TITLE
Looking for more details on "Single-Page" versus having multiple screens

### DISCUSSION THREAD

**Discussion by pricedev** [Budibase/budibase#7856] [General]
Looking for any additional explanation or information on the meaning of a "Single-Page" application vs having multiple screens.  I looked through the docs on screens and that makes sense, but aren't those screens just additional pages, or am I missing something?

What functionality can't be implemented on an additional screen and would therefore require a multi-page application?

Hope that makes sense.

Thanks!

**Comment by melohagan** [comment]
Hi @pricedev 

I would stick to multiple pages as much as possible. The performance of the page can suffer when you have many data providers, repeaters etc!
